integration-tests/src/test/resources/bash-scripts/wdt-create-fmwdomain-onpv.sh

```diff
@@ -0,0 +1,253 @@
+#!/bin/bash
```

Comment:
This is similar to what is already available in  create-fmw-infrastructure-domain/domain-home-on-pv/wdt/create-domain-script.sh.  Can you re-use them instead of maintaining a similar copy.

Reply:
as discussed. this file is renamed to setup_wdt.sh